packages/renderer/src/lib/preferences/SettingsPage.svelte

```diff
@@ -6,8 +6,10 @@ export let title: string;
   <div class="min-w-full px-5 py-4" role="region" aria-label="header">
     <div class="flex flex-row">
       <div class="grow">
-        <p class="capitalize text-xl">{title}</p>
-        <p class="text-sm text-gray-700"><slot name="subtitle"><br /></slot></p>
+        <p class="capitalize text-xl" role="heading" aria-level="1" aria-label="title">{title}</p>
```

Comment:
Imho using `p` for headings is wrong. On the one side, you have a paragraph, which is role dedicated for some purpose, and then you have basically a content in it, that is heading. This should be fixed. 
It should be rewritten into either (and I am not sure what is better) into `h1` or `h2` elements, or into `div` or `span` with a dedicated role `heading` and the level as you put it here.

Reply:
fixed, please review and approve, reference screenshots in PR description